gradle.properties

```diff
@@ -1,5 +1,19 @@
-group=com.uber.sdk
-groupId=com.uber.sdk
-artifactId=rides-android
-githubDownloadPrefix=https://github.com/uber/rides-android-sdk/releases/download/
-version=0.6.2-SNAPSHOT
+GROUP=com.uber.sdk
+VERSION_NAME=0.7.0-SNAPSHOT
+POM_URL=https://developer.uber.com
+
+POM_SCM_URL=https://github.com/uber/rides-android-sdk/
+POM_SCM_CONNECTION=scm:git:git://github.com/uber/rides-android-sdk.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/uber/rides-android-sdk.git
+
+POM_LICENCE_NAME=MIT License
+POM_LICENCE_URL=http://www.opensource.org/licenses/mit-license.php
+POM_LICENCE_DIST=repo
+
+POM_DEVELOPER_ID=tyvsmith
```

Comment:
Nit: for company-wide stuff, we should be using `uber` and `Uber Technologies` for ID and the name. I need to update AutoDispose as well. [Example](https://github.com/square/leakcanary/blob/master/gradle.properties#L15-L16)

Reply:
Cool, since this is already what's being put out in the previous poms, I'll leave it to avoid a bunch of rebases, but I'll update it going forward.

Reply:
:+1: